src/mon/MgrMap.h

```diff
@@ -179,6 +179,18 @@ class MgrMap
     return false;
   }
 
+  bool can_run_module(const std::string &module_name, std::string *error) const
+  {
+    for (const auto &i : available_modules) {
+      if (i.name == module_name) {
+        *error = i.error_string;
+        return i.can_run;
+      }
+    }
+
+    return false;
```

Comment:
shall throw a `std::logic_error` here? as `module` should be in `available_modules`

Reply:
reasonable idea, done.